index.md

```diff
@@ -187,4 +220,47 @@ Some journals, for example, *Annual Reviews* lets the authors retain the right t
 If an article is an adaptation of a previously published article, it must be noted in the cover letter, and major changes noted. 
 Evaluating whether such changes constitute a significant revision will be part of the review process.
 
+## Author Order
+Authors should, to the extent possible, determine the author order
+among themselves.  Each work must have a section describing the actual
+contributions of authors (and of those acknowledged) to provide
+clarity, and journal templates include such a section.  
+<!-- MRS: still need to add that section of the templates.  We should also likely provide more explicit instructions here as to how finely to divide the contributions. -->
+
+However, we acknowledge that institutions evaluating for merit, promotion, tenure
+or other cases may not read this level of detail, so traditional
+notions of author order (first author, corresponding author, etc.) may
+still be relevant and the authors will need to coordinate who should
+occupy these positions.
+
+## Abandoned Documents
+
+By the submission of their paper, document, or materials, all the
+authors consent that if they no longer are willing or able to maintain
+their work, LiveCoMS may assign another individual or individuals to
+do so, with appropriate modifications to the authors list. Authors
+would be given written notice (by e-mail and formal letter) if this
+were to happen and would have a period of six months to respond and/or
+designate a successor before LiveCoMS would do so for them. Typically,
+authors are _implicitly_ giving a right to do this via licensing under
+[Creative Commons - Attribution](https://creativecommons.org/licenses/by/4.0/) or
+similar licenses which give others the right to create derivative
+works (potentially allowing others to ``resurrect'' a document which
```

Comment:
LaTeX formatting here needs to be changed; I think in markdown "implicitly" should be in single asterisks, and "resurrect" should be in regular quotes.

Reply:
done